### DISCUSSION TITLE
InTransaction crypto_in

### DISCUSSION THREAD

**Discussion by topcoderasdf** [eprbell/dali-rp2#92] [Q&A] [answered]
I have a  quick question about the `crypto_in` parameter.
Does `crypto_in` indicate `crypto_in_no_fee` or `crypto_in_with_fee`?

> crypto_in: how much of the given cryptocurrency was acquired with the transaction

Documentation seems to indicate it is `crypto_in_no_fee`, but I wasn't 100% certain.

**Comment by eprbell** [comment, marked as answer]
That's correct: `crypto_in` doesn't contain the fee. I fixed it in the docs (both DaLI and RP2), so that it's clearer.
